ts/components/ConversationListItem.tsx

```diff
@@ -213,7 +214,13 @@ export class ConversationListItem extends React.PureComponent<Props> {
     if (!lastMessage && !isTyping) {
       return null;
     }
-    const text = lastMessage && lastMessage.text ? lastMessage.text : '';
+    let text = lastMessage && lastMessage.text ? lastMessage.text : '';
+
+    // if coming from Rss feed
+    if (lastMessage && lastMessage.isRss) {
```

Comment:
Not familiar with this code, what is lastMessage doing here?
Will you be stripping html from all messages?

Reply:
we're just checking to make sure lastMessage is not undefined, because undefined.isRss will throw

Reply:
lastMessage is the text of the messages under the conversation name in the convo list